docker-compose.local.yml

```diff
@@ -21,7 +21,10 @@ services:
 
   mock:
     platform: ${LOCAL_PLATFORM}
+    image: null # This is necessary to override the image specified in the base file
     build: https://github.com/ministryofjustice/hmcts-common-platform-mock-api.git#main
```

Comment:
Would it not be easier to just temporarily update this one line here to point at the code with the new seed task?

Reply:
At the end was not too bad,
I simplified it, it's just few lines.
To load the seeds in in local you just need:
`docker compose exec mock bundle exec rake mock:demodata:unload mock:demodata:load`